Chrysalis-Papilio/Soen-490-Papilio
# Navbar - Reintegrating Navbar into main

**Navbar**

What to test:
- Verify that the Navbar can be seen upon successful login/signup
- Verify that the buttons on the Navbar are interactable
- Verify that the Navbar button animations play
- Verify that the Navbar buttons open up different empty placeholder fragments

How to do it:
- Simply run the app, log in using whatever method you wish
- Then tap "Account" on the Nav Bar
- Then tap "Activities" on the Nav Bar
- Then tap "Home" on the Nav Bar

Testing:
- run the tests locally using the command ./gradlew connectedAndroidTest
